vic/vic_run/src/func_surf_energy_bal.c

```diff
@@ -749,67 +743,46 @@ func_surf_energy_bal(double  Ts,
 
        Should evapotranspiration be active when the
        ground is only partially covered with snow????
-
-       Use Arno Evap in the exposed soil portion, and/or
-       if LAI is zero.
     *************************************************/
-    if (VEG && !SNOWING && veg_var->fcanopy > 0) {
-        Evap = canopy_evap(layer, veg_var, true,
-                           veg_class, Wdew, delta_t, NetBareRad, vpd,
-                           NetShortBare, Tair, Ra_veg[1], elevation, rainfall,
-                           Wmax, Wcr, Wpwp, frost_fract, root, dryFrac,
-                           shortwave, Catm, CanopLayerBnd);
-        if (veg_var->fcanopy < 1) {
-            for (i = 0; i < options.Nlayer; i++) {
-                transp[i] = layer[i].evap;
-                layer[i].evap = 0.;
-            }
+    for (i=0; i<options.Nlayer; i++) {
```

Comment:
did you already run uncrustify? this line has some formatting issues

Reply:
I'll fix it by hand.  Unfortunately, when I run uncrustify on my system, using the provided config file, it makes a ton of unwanted changes in files that I've never touched.  Not sure why.